src/RuleSetInterface.php

```diff
@@ -30,7 +30,7 @@ public static function create(array $set = array());
      *
      * @param string $rule name of rule
      *
-     * @return array|null
+     * @return array|bool
```

Comment:
`RuleSet` is to resolve rule. one part of resolving is removing rules from sets that were disabled after all.
I see no value in returning disabled rules by RuleSet.

If wanted to validate, it should be validated somewhere else.